test/one/profiler/test/TestProcess.java

```diff
@@ -261,19 +264,43 @@ public Output profile(String args, boolean sudo) throws IOException, TimeoutExce
             throw new IOException("Profiling call failed: " + readFile(PROFERR));
         }
 
-        return readFile(PROFOUT);
+        return readFile(PROFOUT, args.split("\\s+"));
     }
 
     public File getFile(String fileId) {
         return tmpFiles.get(fileId);
     }
 
-    public Output readFile(String fileId) {
+    public Output readFile(String fileId, String... args) {
```

Comment:
`readFile` should return file contents as is. Implicit conversion makes semantics of the method obscure.
Also, `.html` file will not be always a flame graph. For example, one of recent PRs adds heatmaps that also produce output in `.html` format.